### ISSUE TITLE
Error while trying to save randomized italian DPPT roms

### ISSUE THREAD

**Issue by GoogleCodeExporter** [todtim2/universal-pokemon-randomizer#51]
```
What steps will reproduce the problem?
1. Open the randomizer
2. Select options
3. Select Randomize (save)

What is the expected output? What do you see instead?
When i try to save DPPT italian roms, i always get the error "There was an 
unhandled exception trying to save your ROM". The randomizer was working until 
the 1.2.0a version with this these roms. It doesn't matter what option i try to 
select (it makes the error even without select an option).
The randomizer is still work with the other italian rom.

What version of the product are you using? On what operating system?
1.6.0a, on windows 7

Please provide any additional information below.
Attach to the issue there is a zip file with the error that i get while trying 
to randomize the roms.
The first one is made on pearl (I) and with the update moves option, the second 
one is made on diamond (I) with nothing select (only tring to save the rom), 
and the last one is on platinum (I) while used the super_randomizer_race 
default settings.

```

Original issue reported on code.google.com by `tibio...@gmail.com` on 13 May 2014 at 5:23

Attachments:
- [Errors.zip](https://storage.googleapis.com/google-code-attachments/universal-pokemon-randomizer/issue-51/comment-0/Errors.zip)

**Comment by GoogleCodeExporter** [comment]
```
Acknowledged this bug, it's actually been around since 1.5.0 but the new error 
reporting has made it more visible.

Until this is fixed, you can try using a version before 1.5.0 or play on 
English.
```

Original comment by `dabomstew` on 14 May 2014 at 4:05
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Clarification: support for pretty much every non-English gen4 game is broken 
since 1.5.0. Non-English games outside of gen4 should still work, as should 
gen4 English games. The next release will fix this problem.
```

Original comment by `dabomstew` on 14 May 2014 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
Thank you for accepted the issue, also sorry for my bad english.
```

Original comment by `tibio...@gmail.com` on 14 May 2014 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.6.1
```

Original comment by `dabomstew` on 16 May 2014 at 10:36
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
please, there is the same error with the platinum spanish version

```

Original comment by `perrogra...@gmail.com` on 8 Jul 2015 at 5:27
